dockstore-webservice/src/test/java/io/dockstore/webservice/languages/CWLHandlerTest.java

```diff
@@ -101,4 +105,58 @@ public void testURLFromEntry() {
         Assert.assertEquals("https://www.dockstore.org/containers/registry.hub.docker.com/foo/bar", handler.getURLFromEntry("foo/bar", toolDAO));
         Assert.assertEquals("https://www.dockstore.org/containers/registry.hub.docker.com/foo/bar", handler.getURLFromEntry("foo/bar:1", toolDAO));
     }
+
+    @Test
+    public void testGetContentWithMalformedDescriptors() throws IOException {
+        CWLHandler cwlHandler = new CWLHandler();
+
+        // create and mock parameters for getContent()
+        final Set<SourceFile> emptySet = Collections.emptySet();
+        final ToolDAO toolDAO = Mockito.mock(ToolDAO.class);
+        when(toolDAO.findAllByPath(Mockito.anyString(), Mockito.anyBoolean())).thenReturn(null);
+
+        // expect parsing error
+        File cwlFile = new File(ResourceHelpers.resourceFilePath("brokenCWL.cwl"));
+        try {
+            cwlHandler.getContent("/brokenCWL.cwl", FileUtils.readFileToString(cwlFile, StandardCharsets.UTF_8), emptySet,
+                LanguageHandlerInterface.Type.TOOLS, toolDAO);
+        } catch (CustomWebApplicationException e) {
+            Assert.assertEquals(e.getResponse().getStatus(), 400);
```

Comment:
swap parameters, it's assertEquals(expected, actual).  Here and other places

Reply:
[Swapped order](https://github.com/dockstore/dockstore/blob/c5277c33e2a3bf271f3be4e33b742fb646b1a076/dockstore-webservice/src/test/java/io/dockstore/webservice/languages/CWLHandlerTest.java#L126), thanks for the catch, I've mixed these up before so I'll make a note to double check my parameter orderings.